app/Console/Kernel.php

```diff
@@ -16,6 +16,11 @@ class Kernel extends ConsoleKernel
     protected function schedule(Schedule $schedule)
     {
         $schedule->command('weatherforecast:process')->dailyAt('01:00');
+        $schedule->command('humidityProcess:save')->monthly();
```

Comment:
I changed $today to $todayMinusOneMonth because we asks one month periods and till is not end of the month the work gonna fail anyway,,, I run the command twice because at first it finds the nulls of began_at and it finish the job...When i run it for the second time it finds  the nulls of finsihed_at and finish the job

Reply:
I separated the two jobs in two different command so it has better fit to single responsibility principle and i dont repeat myself :)